templated_output/Arkansas_Nuclear_1.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" standalone="no" ?>
 
-<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
+<simulation>
```

Comment:
This isn't right. The reactor xml files should not have simulation tags in them. Each represents just one small part of a simulation.